lib/node_modules/@stdlib/stats/base/snanmeanwd/benchmark/benchmark.js

```diff
@@ -21,16 +21,30 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var randu = require( '@stdlib/random/base/randu' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
 var pow = require( '@stdlib/math/base/special/pow' );
-var Float32Array = require( '@stdlib/array/float32' );
+var uniform = require( '@stdlib/random/base/uniform' );
+var bernoulli = require( '@stdlib/random/base/bernoulli' );
+var filledarrayBy = require( '@stdlib/array/filled-by' );
 var pkg = require( './../package.json' ).name;
 var snanmeanwd = require( './../lib/snanmeanwd.js' );
 
 
 // FUNCTIONS //
 
+/**
+* Returns a random value or `NaN`.
+*
+* @private
+* @returns {number} random number or `NaN`
```

Comment:
```suggestion
* Returns a random value.
*
* @private
* @returns {number} random number
```

Applies here and elsewhere in this PR.